api/environments/environment_controller.go

```diff
@@ -862,6 +872,65 @@ func RestartApplication(accounts models.Accounts, w http.ResponseWriter, r *http
 	radixhttp.JSONResponse(w, r, "Success")
 }
 
+// RestartComponent Restarts job
+func RestartComponentAuxiliaryResource(accounts models.Accounts, w http.ResponseWriter, r *http.Request) {
+	// swagger:operation POST /applications/{appName}/environments/{envName}/components/{componentName}/aux/{auxType}/restart component restartComponentAuxiliaryResopurce
```

Comment:
Typo in `restartComponentAuxiliaryResopurce`

Reply:
fixed